playwright/network.py

```diff
@@ -97,25 +99,40 @@ async def abort(self, errorCode: str = None) -> None:
 
     async def fulfill(
         self,
-        status: int = None,
+        status: int = 200,
         headers: Dict[str, str] = None,
         body: Union[str, bytes] = None,
+        path: Union[str, Path] = None,
         contentType: str = None,
     ) -> None:
         params = locals_to_params(locals())
-        if contentType:
-            if headers is None:
-                headers = {}
-            headers["Content-Type"] = contentType
-            del params["contentType"]
-        if headers:
-            params["headers"] = serialize_headers(headers)
+        length = 0
         if isinstance(body, str):
             params["body"] = body
             params["isBase64"] = False
+            length = len(body)
         elif isinstance(body, bytes):
             params["body"] = base64.b64encode(body).decode()
             params["isBase64"] = True
+            length = len(body)
+        elif path:
+            file_content = Path(path).read_bytes()
+            params["body"] = base64.b64encode(file_content).decode()
```

Comment:
This Python contract is weird. I'd explicitly say decode('ascii') to win the reader's trust.